pkg/notifications/shoutrrr.go

```diff
@@ -28,18 +28,28 @@ type shoutrrrTypeNotifier struct {
 
 func newShoutrrrNotifier(c *cobra.Command, acceptedLogLevels []log.Level) t.Notifier {
 	flags := c.PersistentFlags()
-
 	urls, _ := flags.GetStringArray("notification-url")
+	template := getShoutrrrTemplate(c)
+	return createSender(urls, acceptedLogLevels, template)
+}
+
+func newShoutrrrNotifierFromURL(c *cobra.Command, url string, levels []log.Level) t.Notifier {
+	template := getShoutrrrTemplate(c)
+	return createSender([]string{url}, levels, template)
+}
+
+func createSender(urls []string, levels []log.Level, template *template.Template) t.Notifier {
 	r, err := shoutrrr.CreateSender(urls...)
+	fmt.Println(urls)
```

Comment:
I don't think you are supposed to be here.

Reply:
Correct. Removed.